content/chat/rooms/presence.textile

```diff
@@ -300,13 +296,12 @@ The following options can be set when "creating a room":/docs/chat/rooms#create
 blang[javascript].
 
   |_. Property |_. Description |_. Default |
-  | enter | Set whether the client has permissions to enter the presence set. Calling @presence.enter()@ is still required. | true |
-  | subscribe | Set whether the client has permissions to subscribe to the presence set. Calling @presence.subscribe()@ is still required. | true |
+  | enableEvents | Set whether the client has permissions to subscribe to the presence set. Calling @presence.subscribe()@ is still required. | true |
 
 blang[react].
 
   |_. Property |_. Description |_. Default |
-  | subscribe | Set whether the client has permissions to subscribe to the presence set. Calling @presence.subscribe()@ is still required. | true |
+  | enableEvents | Set whether the client has permissions to subscribe to the presence set. Calling @presence.subscribe()@ is still required. | true |
```

Comment:
Small thing but this is the same as the JS one, so you could just add `react` to the existing blang and remove this one.